lib/idx/interact.ts

```diff
@@ -35,6 +35,20 @@ export interface InteractResponse {
   meta: IdxTransactionMeta;
 }
 
+/* eslint-disable camelcase */
+export interface InteractParams {
```

Comment:
remove export to keep this type private

Reply:
this type is used in a few other places: https://github.com/okta/okta-auth-js/blob/jp-OKTA-417473/lib/types/api.ts#L298